app/javascript/src/components/Dashboard/Notes/List/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import EmptyNotesListImage from "images/EmptyNotesList";
+import { useTranslation } from "react-i18next";
+
+import EmptyState from "components/commons/EmptyState";
+import { SINGULAR } from "constants";
+
+import Card from "./Card";
+
+const List = ({ notes = [], setShowNewNotePane }) => {
+  const { t } = useTranslation();
+
+  const isNotesEmpty = notes.length === 0;
```

Comment:
This variable is kinda unnecessary because the statement itself is pretty self-explanatory.